tests/span_tests.cpp

```diff
@@ -1180,7 +1180,7 @@ TEST(span_test, from_array_constructor)
  #endif
      {
          auto f = [&]() {
-             const span<int, 4> _s4 = {arr2, 2};
+             span<int, 4> _s4{arr2, 2};
```

Comment:
Why remove top-level `const`? (Doesn't really matter since the variable is discarded, but `s2` above remains `const`.)

Reply:
No real reason, I was experimenting with something and forgot to add it back. It's fixed now.